ONSdigital/design-system
# Add external link macro for inline SVGs

### What is the context of this PR?
Fixes #835 
Adds inline svgs for external links and adds design system forum missing links

### How to review 
- Tests pass
- View code
- Check all pages and links display correctly
- Check that link icon is correct colour when focused on
